client/client_test.go

```diff
@@ -0,0 +1,89 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/getamis/sirius/log"
+)
+
+func ExampleUncleByNumberAndPosition() {
```

Comment:
Why do we need the example check?

Reply:
Just introduce how to use the API and we can debug with this quickly.

Reply:
The [`Output`](https://github.com/getamis/eth-indexer/pull/45/files#diff-fce84029d346504e44323c30d21a3808R27) have double commented `////`, so the example will never be executed.